api/utils.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/user/userentity"
+)
+
+// user represents a user in the users.txt file
+// The format of each line in the file is:
+// (authentication_type:)?email password
+func parseUserFromLine(u user) (username, email, password, authenticationType string) {
```

Comment:
The more I look at this the more I am wondering whether this logic should be part of `getUserCredentials`. Would that make sense or are there further complexities?

Reply:
Yeah that's probably a better place for it: 
- Logic refactor: https://github.com/mattermost/mattermost-load-test-ng/pull/742/commits/a6798195fc51d26b9dc11b024c2b25ee5ffd9064
- Tests: https://github.com/mattermost/mattermost-load-test-ng/pull/742/commits/0fc2bc12c5b7253372965237b46c1f0c2446b217